src/NServiceBus.AcceptanceTests/Basic/When_deferring_to_non_local.cs

```diff
@@ -13,7 +13,13 @@ public void Message_should_be_received()
             var context = new Context();
 
             Scenario.Define(context)
-                    .WithEndpoint<Endpoint>(b => b.Given((bus, c) => bus.Send(new MyMessage(), new SendOptions(delayDeliveryFor: TimeSpan.FromSeconds(3)))))
+                    .WithEndpoint<Endpoint>(b => b.Given((bus, c) =>
+                    {
+                        var options = new SendOptions();
+
+                        options.DelayDeliveryWith(TimeSpan.FromSeconds(3));
```

Comment:
Definitely a good idea, we've talked about making time "pluggable" in the core to allow users to test long running things more easily. I'd say that this is out of scope for this pull though